tests/pact/keysets.repl

```diff
@@ -203,3 +203,54 @@
  "Scoped acquire of O succeeds"
  "Capability acquired"
  (test-capability (O)))
+
+;;
+;; keyset formats
+;;
+
+(env-exec-config ["EnforceKeyFormats"])
+(env-data
+ { 'bad: ['foo]
+ , 'short: ["12440d374865bdf0a3349634a70d1317fc279e7e13db98f2199ac5e7378975"]
```

Comment:
Should we add one character to this short test to cover the largest possible number of failure cases?

Reply:
This PR has merged but I manually checked and that case works.